sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_models.py

```diff
@@ -857,22 +846,22 @@ def reuse_key(self):
         return self._reuse_key
 
     @property
-    def curve(self):
+    def key_curve_name(self):
         # type: () -> KeyCurveName
         """Elliptic curve name.
 
         :rtype: ~azure.keyvault.certificates.KeyCurveName
         """
-        return self._curve
+        return self._key_curve_name
 
     @property
-    def ekus(self):
+    def enhanced_key_usage(self):
         # type: () -> list[str]
         """The enhanced key usage.
 
         :rtype: list[str]
         """
-        return self._ekus
+        return self._enhanced_key_usage
 
     @property
     def key_usage(self):
```

Comment:
These property names are right, though - only the types need to change to differentiate them from those in Keys if used together.

Reply:
We talked about this in the meeting yesterday and for consistency sake @schaabs said this should be enhanced_key_usage instead of ekus

Reply:
Yeah there's no need to differentiate in python because you can treat the two types as equivalent.